src/language-js/utils.js

```diff
@@ -978,24 +978,11 @@ function isTSXFile(options) {
   return options.filepath && /\.tsx$/i.test(options.filepath);
 }
 
-function shouldPrintComma(options, level) {
-  level = level || "es5";
-
-  switch (options.trailingComma) {
-    case "all":
-      if (level === "all") {
-        return true;
-      }
-    // fallthrough
-    case "es5":
-      if (level === "es5") {
-        return true;
-      }
-    // fallthrough
-    case "none":
-    default:
-      return false;
-  }
+function shouldPrintComma(options, level = "es5") {
```

Comment:
Can you add JSDoc comments to clarify the type of `level`?

Reply:
Sorry, I don't know how to do it, can you help me?

Reply:
Done https://github.com/prettier/prettier/pull/8752/commits/819d58caff169f321e5aeb2c1e6fbd7adf9c4d9e

Reply:
Should it be optional?

Reply:
Yes! Thank you I missed